proto/redpanda/core/admin/v2/shadow_link.proto

```diff
@@ -260,6 +260,17 @@ message TopicMetadataSyncOptions {
     // If this is true, then only the properties listed in
     // `synced_shadow_topic_properties` will be synced.
     bool exclude_default = 4;
+    // Start at the earliest offset in the partition
```

Comment:
Based on docs team we want these to have periods at the end.

Reply:
done.